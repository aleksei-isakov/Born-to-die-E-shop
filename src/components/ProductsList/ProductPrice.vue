<template>
  <div class="product-price"><slot></slot></div>
</template>

<script>
export default {
  name: 'ProductPrice'
};
</script>

<style lang="scss" scoped>
@import '@/scss/CustomVariables.scss';
.product-price {
  color: black;
  font-size: 1.5rem;
}
.crossed {
  text-decoration: line-through;
  color: $font-color-subtitle;
  font-size: 14px;
  margin-left: 10px;
}
</style>
