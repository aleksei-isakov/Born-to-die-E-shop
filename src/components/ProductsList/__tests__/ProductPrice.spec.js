import { mount } from '@vue/test-utils';
import ProductPrice from '@/components/ProductsList/ProductPrice.vue';

describe('ProductPrice', () => {
  it('renders correctly', () => {
    const wrapper = mount(ProductPrice);

    expect(wrapper.element).toMatchSnapshot();
  });
});
