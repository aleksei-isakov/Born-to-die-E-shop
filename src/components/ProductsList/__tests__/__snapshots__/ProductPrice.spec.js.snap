// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductPrice renders correctly 1`] = `
<div
  class="product-price"
/>
`;
